TF-12000 - Bitbucket Server Deprecation (#574)
hashicorp/terraform-docs-common: website/docs/cloud-docs/vcs/index.mdx

### SSH Keys

> For most supported VCS providers, Terraform Cloud does not need an SSH key — it can do everything it needs with the provider's API and an OAuth token. The exceptions are Azure DevOps Server and Bitbucket Data Center, which require an SSH key for downloading repository contents. The setup instructions for  [Azure DevOps Server](/terraform/cloud-docs/vcs/azure-devops-server) and [Bitbucket Data Center](/terraform/cloud-docs/vcs/bitbucket-data-center) include this step.

Reviewer: ```suggestion
For most supported VCS providers, Terraform Cloud does not need an SSH key. This is because Terraform can do everything it needs with the provider's API and an OAuth token. The exceptions are Azure DevOps Server and Bitbucket Data Center, which require an SSH key for downloading repository contents. Refer to the setup instructions for  [Azure DevOps Server](/terraform/cloud-docs/vcs/azure-devops-server) and [Bitbucket Data Center](/terraform/cloud-docs/vcs/bitbucket-data-center) for details.
```
